### ISSUE TITLE
Accessibility: MAS13: Zoom: RHS pane contents are missing when zoomed to 200%

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#27]
#A11yMAS
#MAS13
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
**Scenario1:**
1.Open https://code.visualstudio.com/docs/introvideos/overview in edge,zoom the page to 200%
2.Navigate to the RHS pane beside the "Introductory Videos" below the header of the page.
3.Observe the RHS pane.[screenshot]

**Scenario2:**
1.Open https://code.visualstudio.com/docs/editor/codebasics in edge,zoom the page to 200%
2.Navigate to the RHS pane beside the "The Basics of Visual Studio Code" below the header of the page.
3.Observe the RHS pane.[screenshot]

**Note: same issue observed for below links**
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/runtimes/nodejs
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
**Scenario1:**
Observe the RHS pane of the page ,the **In this article** text and **Tweet button**  are missing when zoomed to 200% but clearly visible when zoomed to normal (100%).

**Issue repro in below resolutions:
Resolution: 1366x768
Resolution: 1280x1024**

**Scenario2:**
Observe the RHS pane of the page ,the links under  **In this article** and **Tweet button** are missing when zoomed to 200% but clearly visible when zoomed to normal (100%).

**Issue repro in below resolutions:
Resolution: 1366x768
Resolution: 1280x1024**

**Expected Result:**
The contents should not be missed when zoomed to 200%.

**User Impact:**
Users with low vision and people who need cognitive assistance will not be able identify the missing interactive elements.

**Screenshots:**
[RHS pane screen shots.zip](https://github.com/wiproaccess/vscode-website/files/689096/RHS.pane.screen.shots.zip)

**Comment by chrisdias** [comment]
dupe of https://github.com/wiproaccess/vscode-website/issues/23
